app/logging/logging_config.py

```diff
@@ -104,10 +110,11 @@ def customize_logging(
             enqueue=True,
             backtrace=False,
             level=LOGLEVEL_ERROR,
+            serialize=SERIALIZE,
         )
 
         # Return the logger bound with additional context
```

Comment:
No longer returning additional context, or just get rid of comment

```suggestion
        # Return the logger
```